scripts/lstchain_data_muon_analysis_dl1.py

```diff
@@ -0,0 +1,132 @@
+import argparse
+import sys, os
+import numpy as np
+from ctapipe.image.muon.features import ring_containment 
+from ctapipe.image.muon.features import ring_completeness
+from ctapipe.image.muon.features import npix_above_threshold
+from ctapipe.image.muon.features import npix_composing_ring
+from ctapipe.image.muon.muon_integrator import MuonLineIntegrate
+from ctapipe.image.cleaning import tailcuts_clean
+from ctapipe.instrument import CameraGeometry
+from ctapipe.io.hdf5tableio import HDF5TableReader
+from astropy import units as u
+
+from lstchain.image.muon import analyze_muon_event
+from lstchain.io.io import dl1_params_lstcam_key, dl1_images_lstcam_key
+
+from astropy.table import Table
+import pandas as pd
+import tables
+
+'''
+Script to perform the analysis of muon events.
+To run it, type:
+
+python lstchain_data_muon_analysis_dl1.py 
+--input_file dl1_Run01566_0322.h5 
+--output_file Data_table.fits 
+'''
+
+parser = argparse.ArgumentParser()
+
+# Required arguments
+parser.add_argument("--input_file", help = "Path to fits.fz data file.",
```

Comment:
Make clear in the help that this script takes dl1(a) .h5 files as input. The rest seems fine to me.